sdks/java/core/src/main/java/org/apache/beam/sdk/testing/TestPipelineOptions.java

```diff
@@ -50,6 +52,14 @@
   Long getTestTimeoutSeconds();
   void setTestTimeoutSeconds(Long value);
 
+  @Default.Boolean(false)
+  @Internal
+  @Hidden
+  @org.apache.beam.sdk.options.Description(
+      "Indicates whether this is an automatically-run unit test.")
```

Comment:
...as opposed to? (what else is TestPipelineOptions used for, if not unit tests?)